rectangle_vehicles.py

```diff
@@ -1,8 +1,9 @@
-from google.transit import gtfs_realtime_pb2
-from google.protobuf import json_format
+import os
```

Comment:
Do we still need this file at all....? Wondering if it's superseded by all the processing in `data-infra`

Reply:
Probably not, I will mark as deprecated/unneeded without deleting to keep this PR smaller